README.md

```diff
@@ -78,6 +78,9 @@ List of resources on scholarly data analysis ranging from datasets, papers, and
 * [OSF API (Open Science Framework)](https://developer.osf.io/)
 * [Digital tools for researchers](http://connectedresearchers.com)
 * [Fatcat - versioned, publicly-editable catalog of research publications](https://fatcat.wiki/)
+* [Microsoft Academic Knowledge Graph](http://ma-graph.org/)
```

Comment:
Can you change the name of Microsoft Academic Knowledge Graph to Microsoft Academic Graph - RDF dump as we already have MAG at the top. Another option is to add a separate link next to MAG with title RDF triples.